code/modules/organs/external/_external.dm

```diff
@@ -1009,7 +1009,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 		W.germ_level = 0
 	return rval
 
-/obj/item/organ/external/proc/clamp()
+/obj/item/organ/external/proc/clampwound()
```

Comment:
Now, to be pedantic. `clamp_wound` would have been a fancier and better rename.
```suggestion
/obj/item/organ/external/proc/clamp_wound()
```
Don't forget to do this to the three lines in `code/modules/surgery/generic.dm` as well.